src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -252,3 +252,15 @@ def sdk_no_wait(no_wait, func, *args, **kwargs):
     if no_wait:
         kwargs.update({'raw': True, 'polling': False})
     return func(*args, **kwargs)
+
+
+def open_page_in_browser(url):
+    if sys.platform.lower() == 'darwin':
+        # handle 2 things:
+        # a. On OSX sierra, 'python -m webbrowser -t <url>' emits out "execution error: <url> doesn't
+        #    understand the "open location" message"
+        # b. Python 2.x can't sniff out the default browser
```

Comment:
This code is addressing these issues, or these need to be addressed?

Reply:
It is addressing these issues. The code was moved over from the appservice module since core module also needs to launch a browser now.